libmate-panel-applet/mate-panel-applet.c

```diff
@@ -214,18 +227,18 @@ mate_panel_applet_set_flags (MatePanelApplet      *applet,
 		g_variant_builder_init (&invalidated_builder, G_VARIANT_TYPE ("as"));
 
 		g_variant_builder_add (&builder, "{sv}", "Flags",
-				       g_variant_new_uint32 (applet->priv->flags));
+				       g_variant_new_uint32 (priv->flags));
 
-		g_dbus_connection_emit_signal (applet->priv->connection,
-					       NULL,
-					       applet->priv->object_path,
-					       "org.freedesktop.DBus.Properties",
-					       "PropertiesChanged",
+		g_dbus_connection_emit_signal (priv->connection,
+		                               NULL,
+		                               priv->object_path,
+		                               "org.freedesktop.DBus.Properties",
+		                               "PropertiesChanged",
 					       g_variant_new ("(sa{sv}as)",
-							      MATE_PANEL_APPLET_INTERFACE,
-							      &builder,
-							      &invalidated_builder),
-					       &error);
+		                                              MATE_PANEL_APPLET_INTERFACE,
+		                                              &builder,
+		                                              &invalidated_builder),
+		                               &error);
```

Comment:
It is OK to fix the alignment here if you already touch this code :)

Reply:
done